docs/en/observability/profiling-upgrade.asciidoc

```diff
@@ -5,6 +5,16 @@
 <titleabbrev>Upgrade</titleabbrev>
 ++++
 
+[discrete]
+[[profiling-upgrade-known-issues]]
+== Known issues
+
+If you created your {ecloud} deployment starting from a 7.x version or earlier, and you have upgraded it to 8.9.0, 8.9.1 or 8.9.2, you may encounter a problem while trying to enable Universal Profiling.

```

Comment:
Should this be

"and you have upgraded it to 8.9.0, 8.9.1 or 8.9.2"

-> 

"and you are currently upgrading it to 8.9.0, 8.9.1 or 8.9.2"